addons/l10n_in/data/account_tax_template_data.xml

```diff
@@ -1231,4 +1071,342 @@ if tax &gt; result:result=tax</field>
         <field name="sequence">0</field>
         <field name="children_tax_ids" eval="[(6, 0, [ref('sgst_purchase_2_5_rc'),ref('cgst_purchase_2_5_rc'),])]"/>
     </record>
+
+    <record id="non_gst_supplies_purchase" model="account.tax.template">
```

Comment:
We're adding a bunch of new taxes. It's not the place to do this. Remove them from your PR, and tell JVA to do that in his branch, not here.